playbook/app/pb_kits/playbook/utilities/globalProps.ts

```diff
@@ -130,7 +130,7 @@ export type GlobalProps = AlignContent & AlignItems & AlignSelf &
   BorderRadius & Cursor & Dark & Display & DisplaySizes & Flex & FlexDirection &
   FlexGrow & FlexShrink & FlexWrap & JustifyContent & JustifySelf &
   LineHeight & Margin & MaxWidth & NumberSpacing & Order & Padding &
-  Position & Shadow & ZIndex & { hover?: string };
+  Position & Shadow  & ZIndex & { hover?: string };
```

Comment:
Extra space here - but other than that the file didn't change so you can remove it.